e2e/_suites/metricbeat/README.md

```diff
@@ -0,0 +1,102 @@
+[![Build Status](https://beats-ci.elastic.co/buildStatus/icon?job=e2e-tests%2Fe2e-testing-integrations-daily)](https://beats-ci.elastic.co/job/e2e-tests/job/e2e-testing-integrations-daily/)
```

Comment:
Wdyt about propagating the build result from the downstream job to the scheduled one?

Reply:
Yep, I didn't see your comment and I commented the same in `e2e/_suites/helm/README.md` :)

Reply:
It's a matter of adding `propagate: true` to the trigger, right? What if the `wait` is set to false? Will Jenkins update the job status after downstream job finishes?

Reply:
IIRC, propagate will force the wait, but no the other way around

I'd say using the downstream build badge gets some other benefits:
- one worker only for the running, aka, the upstream does not need to block the worker to wait for the downstream build.
- parenstream does not need to populate the test results from the downstream.
- easy to debug what's going in the logs

Reply:
But how to know if the proper build for the scheduled build is green? I can imagine a scheduled execution failing, say it for Helm, and after that the metricbeat suite triggers a green build. Then the Helm badge will point to the status of a green downstream build, which could be misleading

Reply:
@elastic/observablt-robots I'm going to remove the badges, merge, and open a discussion around badges and upstream/downstream jobs.

Thanks!